### ISSUE TITLE
Column in message list not saved when Thunderbird exits

### ISSUE THREAD

**Issue by thistlillo** [mlazdans/qnote#27]
Hello, I am using Thunderbird 102.0.3 (64-bit) on Windows 10 with QNote version 0.11.2. Thank you for this fantastic add-on.

I have a minor issue with it. When I add a column for QNote in the message list, this column is not kept when I close and re-open Thunderbird. I tried to apply the column layout to all the folders, children folders, etc. but whatever I do the column for QNote disappears when I quit Thunderbird.

It would be great if you could look into this issue.

**Comment by mlazdans** [comment]
Can't reproduce. What layout are you using?

**Comment by thistlillo** [comment]
In the menu, "View" -> "Layout", three items are ticked: Classic View, Folder pane, Message pane.

**Comment by mlazdans** [comment]
Exactly as mine. Not sure what is going on there and nobody else has complained. Have you tried first moving the column and then restarting TB?

**Comment by thistlillo** [comment]
Yes, I did, but nothing changed. I noticed an error message in the developer log when the QNote column is empty. The error disappears when I add a new QNote, but this has no effect on the persistence of the column, that keeps disappearing when I close Thunderbird.

**Comment by thistlillo** [comment]
Hello, today these messages appeared in the error console:
```
WebExtensions: qapp.WindowObserver.observe() domwindowopened about:blank [qapp.js:85](jar:file:///C:/Users/User/AppData/Roaming/Thunderbird/Profiles/km8anhr3.default-release/extensions/qnote@dqdp.net.xpi!/api/qapp.js)
QNoteColumnHandler.attachToWindow() - attaching... [QNoteColumnHandler.js:57:17](resource://qnote/modules/QNoteColumnHandler.js)
QNoteColumnHandler.attachToWindow() - not attachable! [QNoteColumnHandler.js:65:18](resource://qnote/modules/QNoteColumnHandler.js)
WebExtensions: KeyboardHandler.addTo() - attaching... [qapp.js:29](jar:file:///C:/Users/User/AppData/Roaming/Thunderbird/Profiles/km8anhr3.default-release/extensions/qnote@dqdp.net.xpi!/api/qapp.js)
WebExtensions: KeyboardHandler.addTo() - not attachable
```

**Comment by mlazdans** [comment]
Closed due no input
